docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -165,6 +165,10 @@ The following core load shed points are supported:
     - Envoy will send a ``GOAWAY`` while processing HTTP2 requests at the codec
       level which will eventually drain the HTTP/2 connection.
 
+  * - envoy.load_shed_points.hcm_ondata_creating_codec
+    - Envoy will close the connections before creating codec if the system is
```

Comment:
Might be useful to expand a bit more on when this runs for end users e.g. this is once / connection

Reply:
Done, thanks!